AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -253,3 +229,18 @@ - (NSString *)_recursiveDescriptionForLayout:(ASLayout *)layout level:(NSUIntege
 }
 
 @end
+
+ASLayout *ASCalculateLayout(id<ASLayoutable> layoutable, const ASSizeRange sizeRange, const CGSize parentSize)
+{
+  ASDisplayNodeCAssertNotNil(layoutable, @"Not valid layoutable passed in.");
+  
+  return layoutable ? [layoutable layoutThatFits:sizeRange parentSize:parentSize] : nil;
```

Comment:
you might be able to rewrite the return a bit shorter `return layoutable ?: [layoutable layoutThatFits:sizeRange parentSize:parentSize]`